packages/office-ui-fabric-react/src/components/Nav/examples/Nav.Basic.Example.scss

```diff
@@ -3,7 +3,7 @@
 :global {
 
   .ms-NavExample-LeftPane {
-    width: 208px;
+    width: 207px;
```

Comment:
Could you add a comment to explain this value?

Reply:
Not sure this is necessary change.  I just make sure the test leftNav have the same width as current SPO page leftNav.

Reply:
this change may or may not needed but be nice to use real word nav width here.

Reply:
The designers usually snap everything to a 4px grid, so it's odd that SPO would have a width of 207px here. Maybe it's 208px including the border? But this is using `box-sizing: border-box` so the width should be inclusive of the border.

Reply:
I revert this back there is no need to make this change.  I only saw from devtools that real nav seems 207 but I don't need to change the example here